onset_maproom/calc.py

```diff
@@ -247,20 +400,59 @@ def seasonal_onset_date(
     dry_spell_search,
     time_coord="T",
 ):
-    """Function reproducing `Ingrid onsetDate function <http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate>`_
-    combining a function that groups data by season
-    and a function that search for an onset date
-
-    Parameters
-    ----------
-    daily_rain : DataArray
-                 Array of daily rainfall totals.
-    search_start_day : int
-                 The day part (1-31) of the date to start scanning for onset date.
-    search_start_month : int
-                 The month part (1-12) of the date to start scanning for onset date.
-    """
+    """ Computes yearly seasonal onset dates from daily soil moisture data.
+
+        Function reproducing Ingrid onsetDate function 
+        '<http://iridl.ldeo.columbia.edu/dochelp/Documentation/details/index.html?func=onsetDate>`_
```

Comment:
Put that in Notes

Reply:
where is this notes you speak of?

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html#notes

I also used it in the PR I mentioned earlier.